R/roxygen2_templates.R

```diff
@@ -0,0 +1,30 @@
+# nocov start
```

Comment:
See https://roxygen2.r-lib.org/articles/reuse.html?q=template#superseded

`man-roxygen` folder is a very old way of placing the template, if we want to keep using templates we should move to `man/roxygen` folder

Reply:
@averissimo should we create a separate issue?

Reply:
For other repos yes, although it's a low priority until it's really deprecated (not only twice superseded)

For this one, depends on this PR and how we deal with `@param decorators` if we keep as is, the change can tag along.